setup.cfg

```diff
@@ -1,14 +1,46 @@
 [flake8]
-select = E,F,W,N,D
-ignore = W503,E126,E133,D107
-exclude = .git,__pycache__,sphinx-doc/conf.py,build,dist
+# include messages from:
+select =
+    E,F,W, # flake8
+    N, # pep8-naming
+    D, # flake8-docstrings
+    RST # flake8-rst-docstrings
+ignore =
+    # line breaks should occur before the binary operator
+    W503,
+    # allow occasional use of uppercase variable and argument names (e.g. N)
+    N806,N803,
+    # match yapf formatting style
+    E126,E133,
+    # do not document __init__ separately from the class
+    D107
+exclude = .git,
+    __pycache__,
+    build,
+    hoomd/extern,
```

Comment:
FYI, running bumpversion will strip all the comments in `setup.cfg`. Not sure if this affects HOOMD.

Reply:
I use a separate `.bumpversion.cfg` to mitigate this issue. These comments will persist.